dlt/common/schema/typing.py

```diff
@@ -155,6 +155,14 @@ class NormalizerInfo(TypedDict, total=True):
     new_table: bool
 
 
+class TMergeConfig(TypedDict, total=False):
+    strategy: Optional[TLoaderMergeStrategy]
+    validity_column_names: Optional[List[str]]
```

Comment:
Should it allow duplicates column names?

Reply:
It shouldn't, didn't think of that. Added validity column name checking in https://github.com/dlt-hub/dlt/pull/1168/commits/30bb2e0e9fff508a76c9cea56aed997c849cf377. An exception is raised if a configured validity column name appears in the data.